pkg/github/commenter/ghcommenter.go

```diff
@@ -0,0 +1,286 @@
+package commenter
+
+import (
+	"errors"
+	"fmt"
+	"strings"
+	"time"
+
+	"github.com/openshift/sippy/pkg/db"
+	"github.com/openshift/sippy/pkg/db/models"
+	"github.com/openshift/sippy/pkg/prowloader/github"
+	log "github.com/sirupsen/logrus"
+	"gorm.io/gorm"
+)
+
+type GitHubCommenter struct {
+	githubClient *github.Client
+	dbc          *db.DB
+	includeRepos map[string]map[string]bool
+	excludeRepos map[string]map[string]bool
+}
+
+const TrtCommentIDKey = `trt_comment_id`
+
+func NewGitHubCommenter(githubClient *github.Client, dbc *db.DB, excludedRepos, includedRepos []string) *GitHubCommenter {
+	ghCommenter := &GitHubCommenter{}
+	ghCommenter.githubClient = githubClient
+	ghCommenter.dbc = dbc
+
+	ghCommenter.excludeRepos = buildOrgRepos(excludedRepos)
+	ghCommenter.includeRepos = buildOrgRepos(includedRepos)
+
+	return ghCommenter
+}
+
+func buildOrgRepos(in []string) map[string]map[string]bool {
+	if in == nil || len(in) < 1 {
+		return nil
+	}
+
+	out := make(map[string]map[string]bool)
+
+	for _, r := range in {
+		ar := strings.Split(r, `/`)
+		var org, repo string
+
+		if len(ar) < 2 {
+			org = `openshift`
+			repo = r
+		} else {
+			org = ar[0]
+			repo = ar[1]
+		}
```

Comment:
Today it would include / exclude openshift/foo and drop the bar.  Can definitely log an error, preference for sticking with the partial match or dropping it all together or outright failing startup?